lib/generators/graphiti/templates/application_resource.rb.erb

```diff
@@ -12,4 +12,21 @@ class ApplicationResource < Graphiti::Resource
   self.adapter = Graphiti::Adapters::ActiveRecord
   self.base_url = Rails.application.routes.default_url_options[:host]
   self.endpoint_namespace = '<%= api_namespace %>'
+
+
+  def self.default_attributes(opts = { })
+    begin
+      unless opts[:from].kind_of?(Class) && opts[:from] <= ApplicationRecord
```

Comment:
Should this be a string instead of a class? Similar to how ActiveRecord's `belongs_to` has a `class_name` option which has a string value.